Care-CureHospital/HospitalMap/Code/Repository/InformationEditRepository.cs

```diff
@@ -0,0 +1,90 @@
+﻿using HospitalMap.WPF.ModelWPF;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Text;
+
+namespace HospitalMap.Repository
+{
+    public class InformationEditRepository
+    {
+        private const String _path = @"C:\Users\PC\Desktop\GIT\Care-CureHospital\Care-CureHospital\HospitalMap\Code\Resources\RoomsInformation.txt";
```

Comment:
You should change path to relative file path instead of using absolute file path.